modules/juce_graphics/native/juce_mac_CoreGraphicsContext.mm

```diff
@@ -983,6 +983,51 @@ void juce_convertColourGradientToDisplayColourSpace (
                                                         screenPtr));
  }
 
+ Image juce_returnImageWithNativeColourSpace (const Image& src,
+                                        const void* screenPtr)
+ {
+     // color spaces
+     auto srgbSpace = CGColorSpaceCreateWithName (kCGColorSpaceSRGB);
+     // display color space based on profile
+     auto nativeSpace = ((NSScreen*)screenPtr).colorSpace.CGColorSpace;
+
+     // to CIImage
+     CGSize imageSize;
+     imageSize.width = src.getWidth();
+     imageSize.height = src.getHeight();
+     const Image::BitmapData inputImage (src, Image::BitmapData::readOnly);
+     // maybe way to use CFDataRef?

```

Comment:
1) Missing `TODO`
2) `CFDataRef` and `NSData` can be used interchangeibly. Why would you prefer using the Core Foundation type?